src/cml.js

```diff
@@ -257,77 +257,38 @@ class CML {
     return await getDriver(this).runnerToken();
   }
 
-  parseRunnerLog(opts = {}) {
+  async parseRunnerLog(opts = {}) {
     let { data } = opts;
-    if (!data) return;
+    if (!data) return [];
 
-    const date = new Date();
+    data = data.toString('utf8');
 
-    try {
-      data = data.toString('utf8');
-
-      let log = {
-        level: 'info',
-        date: date.toISOString(),
-        repo: this.repo
-      };
-
-      if (this.driver === GITHUB) {
-        const id = 'gh';
-        if (data.includes('Running job')) {
-          log.job = id;
-          log.status = 'job_started';
-        } else if (data.includes('completed with result')) {
-          log.job = id;
-          log.status = 'job_ended';
-          log.success = data.includes('Succeeded');
-        } else if (data.includes('Listening for Jobs')) {
-          log.status = 'ready';
-        }
+    const logs = [];
+    const entities = await getDriver(this).runnerParseLogEntities();
+    for (const [entity, rule] of Object.entries(entities)) {
+      const regex = new RegExp(rule);
+      if (regex.test(data)) {
+        const date = new Date();
+        const log = {
+          id: 'dummy',
```

Comment:
id not used/overwritten? Where does `log.job` get set?
ref: https://github.com/iterative/cml/blob/fbec7b00d8f2a6034a8e5fd85eb44e84dd53c1c2/bin/cml/runner.js#L254-L257

Reply:
id is `dummy` right now for every SCM, all the same

Reply:
Its mean to be an id, however that id is only reachable in GL